website/sidebars.js

```diff
@@ -9,129 +9,142 @@
       items: [
         {
-          type: 'doc',
-          id: 'tutorial/introduction',
-          label: 'Introduction',
+          type: "doc",
+          id: "tutorial/introduction",
+          label: "Introduction",
         },
         {
-          type: 'doc',
-          id: 'tutorial/chat-termination',
-          label: 'Chat Termination',
+          type: "doc",
+          id: "tutorial/chat-termination",
+          label: "Chat Termination",
         },
         {
-          type: 'doc',
-          id: 'tutorial/human-in-the-loop',
-          label: 'Human in the Loop',
+          type: "doc",
+          id: "tutorial/human-in-the-loop",
+          label: "Human in the Loop",
         },
         {
-          type: 'doc',
-          id: 'tutorial/code-executors',
-          label: 'Code Executors',
+          type: "doc",
+          id: "tutorial/code-executors",
+          label: "Code Executors",
         },
         {
-          type: 'doc',
-          id: 'tutorial/tool-use',
-          label: 'Tool Use',
+          type: "doc",
+          id: "tutorial/tool-use",
+          label: "Tool Use",
         },
         {
-          type: 'doc',
-          id: 'tutorial/conversation-patterns',
-          label: 'Conversation Patterns',
+          type: "doc",
+          id: "tutorial/conversation-patterns",
+          label: "Conversation Patterns",
         },
         {
-          type: 'doc',
-          id: 'tutorial/what-next',
-          label: 'What Next?',
-        }
+          type: "doc",
+          id: "tutorial/what-next",
+          label: "What Next?",
+        },
       ],
     },
-    {'Use Cases': [{type: 'autogenerated', dirName: 'Use-Cases'}]},
+    { "Use Cases": [{ type: "autogenerated", dirName: "Use-Cases" }] },
     {
-      type: 'category',
-      label: 'User Guide',
+      type: "category",
+      label: "User Guide",
       collapsed: false,
       link: {
-        type: 'generated-index',
-        title: 'User Guide',
-        slug: 'topics',
+        type: "generated-index",
+        title: "User Guide",
+        slug: "topics",
       },
-      items: [{type: 'autogenerated', dirName: 'topics'}]
+      items: [{ type: "autogenerated", dirName: "topics" }],
     },
     {
-      type: 'link',
-      label: 'API Reference',
-      href: '/docs/reference/agentchat/conversable_agent',
+      type: "link",
+      label: "API Reference",
+      href: "/docs/reference/agentchat/conversable_agent",
     },
     {
-      type: 'doc',
-      label: 'FAQs',
-      id: 'FAQ',
+      type: "doc",
+      label: "FAQs",
+      id: "FAQ",
     },
+
     {
-      'type': 'category',
-      'label': 'Ecosystem',
-      'link': {
-        type: 'generated-index',
-        title: 'Ecosystem',
-        description: 'Learn about the ecosystem of AutoGen',
-        slug: 'ecosystem',
+      type: "category",
+      label: "AutoGen Studio",
+      collapsed: true,
+      items: ["autogen-studio/usage", "autogen-studio/faqs"],
```

Comment:
Do you want to use the `generated-index` or put `getting-started` as one of the items?

Reply:
fixed, thanks!